cocos/audio/android/AudioMixerController.cpp

```diff
@@ -50,6 +51,14 @@ AudioMixerController::AudioMixerController(int bufferSizeInFrames, int sampleRat
     _mixingBuffer.buf = memalign(32, _mixingBuffer.size);
     memset(_mixingBuffer.buf, 0, _mixingBuffer.size);
 }
+#elif CC_TARGET_PLATFORM == CC_PLATFORM_OPENHARMONY
+AudioMixerController::AudioMixerController(int sampleRate, int channelCount)
+: _sampleRate(sampleRate), _channelCount(channelCount), _mixer(nullptr), _isPaused(false), _isMixingFrame(false) {
+    ALOGV("In the constructor of AudioMixerController!");
+    int32_t maxBufferSize = 17832; 
```

Comment:
Can you use constants? Also can add comments why 17832?

Reply:
For openharmony platform, buffer size is 4458 in normal latency mode, 240 in fast latency mode. so maxBufferSize = 4458 * 2(channel count) * 2(bit depth).